cmd/quota.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func checkQuota(container Container, result *Result) {
+	if container.Resources.Limits == nil {
+		occ := Occurrence{id: ErrorResourcesLimitNIL, kind: Error, message: "Quota not set, please set it!"}
+		result.Occurrences = append(result.Occurrences, occ)
+		return
+	}
+
+	if container.Resources.Limits.Cpu() == nil || container.Resources.Limits.Cpu().IsZero() {
+		occ := Occurrence{id: ErrorResourcesLimitNIL, kind: Error, message: "CPU quota not set, please set it!"}
```

Comment:
I think we should have 3 different codes for this: `ResourcesLimitsNil`, `ResourcesLimitsCpuNIL` and `ResourcesLimitsMemoryNIL`

Reply:
Ok for 3 different codes, plus two other codes for the parameters case. ResourcesLimitsCpuExceeded and ResourcesLimitsMemoryExceeded.